docs/03_server/04_event-handler/02_basics.md

```diff
@@ -219,9 +219,36 @@ As the  example shows, there is an additional type defined for the context Event
 
 Because the example creates a validation context, the function `validationAck()` is used at the end of the `onValidate` block, and not just `ack()`.
 
-# Auto-generated REST endpoints
+## Handling delete events
 
-As we have mentioned before, all events created in the eventhandler file are automatically exposed as a [REST endpoints](../../../server/integration/rest-endpoints/introduction/).
+There are two ways of handling a delete event:
+
+- you can create a separate class
+- you can use a key set as metadata
```

Comment:
Rather than 'key set', I would use 'index entity'. Set has a specific meaning that doesn't apply here. 

Also, I would place index entity as the first option.